docs/MIGRATION_GUIDE_08.md

```diff
@@ -0,0 +1,238 @@
+# Migration guide matter.js Device API to 0.8
+
+With version 0.8 the matter.js project introduces a new high level API to build devices bases on code generation
+for all Matter 1.1 device types, clusters and functionality. In the past we already generated the code for all 
+cluster definitions and this got now enhanced for all device types. Additionally, the way how own cluster logic can be 
+implemented also needed to be adjusted and got enhanced together with a very flexible way to choose the wanted cluster 
```

Comment:
implemented also needed to be adjusted and was enhanced with a very flexible way to choose the wanted cluster